lib/openstack_query/api/query_api.py

```diff
@@ -114,24 +114,27 @@ def run(
         :param kwargs: keyword args that can be used to configure details of how query is run
             - valid kwargs specific to resource
         """
+        if not cloud_account and not from_subset:
+            raise ParseQueryError(
+                "please provide as a parameter, one of:"
+                "\n\tcloud_account - a cloud domain to run query using openstacksdk"
+                "\n\tfrom_subset - a set of openstack objects"
+            )
 
         if from_subset:
-            logger.debug(
```

Comment:
Why has the `logger.debug` been removed here?

Reply:
I moved it to the Executer Class - which is what we call. 
https://github.com/stfc/st2-cloud-pack/pull/203/files/41436e1d1d9d6058bc666f997e801f119d177821#diff-d8bf356dba2675d5eeb0704114eb88370e704d9be0109a037980d34eb1b44eb6R111